docs/guides/python/serverless-llama.mdx

```diff
@@ -0,0 +1,275 @@
+---
+description: Use Llama model with serverless compute to translate text and store results using Nitric
+tags:
+  - Nitric
+  - API
+  - AI & Machine Learning
+languages:
```

Comment:
rebase and add start_steps. See go realtime guide for example.

Reply:
rebased, but start steps won't work with this repository - it requires them to download llama separately.

Reply:
Could the model be `curl`ed?